.github/workflows/build.yml

```diff
@@ -18,6 +18,22 @@ jobs:
   build:
     name: Build
     runs-on: windows-2022
+    defaults:
+        run:
+          shell: pwsh
+    strategy:
+      matrix:
+        platform: [netframework, net-x86, net-x64]
+        include:
+          - platform: netframework
+            package-name: netframework
+            build-dir: net48
+          - platform: net-x86
+            package-name: win32
+            build-dir: net6.0-windows\win-x86\publish
+          - platform: net-x64
+            package-name: win64
```

Comment:
The package name here should be `net-win64` to preserve the naming convention of the artifacts.